### PR TITLE
Docs for new ACS model

### PR DESCRIPTION
**Pull request by davidbenjamin**

### CODE REVIEW

**Comment by LeeTL1220** [comment]
@davidbenjamin Hey, after talking to @samuelklee it seems that what is in the doc does not really match the (ipython) prototype and (possibly) not what will be implemented in Java.  I will merge this when we update the docs to reflect which approach(es) are in ipython vs. Java vs. future work (stated more explicitly).

File: docs/AllelicCapSeg/ACS-methods.tex [hunk 1]
```diff
@@ -269,6 +273,188 @@ \subsection{Model-comparison test for segment merging} \label{likelihood-based-s
...
+\State Initialize $f_s = \left( N^s_{\rm AA} + N^s_{\rm RR} \right) / \left(N^s_{\rm AR} + N^s_{\rm RA} \right)$
+\State Initialize $\lambda$ randomly and approximately equal to $1$.
+\State Calculate $\alpha_{\rm eff}$ and $\beta_{\rm eff}$ from Equations \ref{alpha_eff} and \ref{beta_eff}.
+\Repeat
+	\State Sample each $f_s$  from the conditional Eq. \ref{conditional_on_f} using Metropolis sampling.
+	\ForAll{sites $j$}
+		\State Sample $\lambda_j$ from the conditional Eq. \ref{conditional_on_lambda} using two-auxiliary-variable slice sampling.
+		\State Update $\alpha_{\rm eff}$ and $\beta{\rm eff}$.
+		\State Sample $z_j$ from the categorical distribution defined by Eq. \ref{likelihood}.
+	\EndFor
```
Comment (LeeTL1220) [inline docs/AllelicCapSeg/ACS-methods.tex]: Don't you mean equation 30 (unlabeled), not 28 (labeled: likelihood)?

File: docs/AllelicCapSeg/ACS-methods.tex [hunk 2]
```diff
@@ -269,6 +273,188 @@ \subsection{Model-comparison test for segment merging} \label{likelihood-based-s
...
+The Bayesian network corresponding to this model is shown in Figure \ref{graphical_model}.
+\begin{figure}
+$
+\begin{array}{c}
+\includegraphics[width=0.8\linewidth]{ACS_model.png} 
+\end{array}
+$
+\label{graphical_model}
+\caption{Graphical model for \ACS} 
+\end{figure}
```
Comment (samuelklee) [inline docs/AllelicCapSeg/ACS-methods.tex]: should there be a \pi_o in this diagram?

**Comment by davidbenjamin** [comment]
@LeeTL1220 Back to you.

**Comment by LeeTL1220** [comment]
Comments were very minor and given on a hardcopy.

**Pull request merged**